crates/hyperqueue/src/client/output/outputs.rs

```diff
@@ -33,6 +35,14 @@ impl FromStr for Outputs {
     }
 }
 
+#[derive(Parser, Clone, Copy)]
```

Comment:
It's not super important, but this does not need to be `Clone` nor `Copy`, let's save some compile time by removing these derives.